src/Aquarius.ONE.ClientSDK/Operations/Sample/SampleApi.cs

```diff
@@ -116,18 +115,83 @@ public async Task<bool> CreateTestGroupAsync(TestAnalyteGroup testGroup)
             try
             {
                 var respContent = await _restHelper.PostRestProtobufAsync(testGroup, endpoint).ConfigureAwait(_continueOnCapturedContext);
-                if (respContent.ResponseMessage.IsSuccessStatusCode)
+                Event(null,
+                    respContent.ResponseMessage.IsSuccessStatusCode
+                        ? CreateLoggerArgs(EnumEventLevel.Trace, "CreateTestGroupAsync Success", respContent.ResponseMessage.StatusCode, watch.ElapsedMilliseconds)
+                        : CreateLoggerArgs(EnumEventLevel.Warn, "CreateTestGroupAsync Failed", respContent.ResponseMessage.StatusCode, watch.ElapsedMilliseconds));
+
+                return respContent.ResponseMessage.IsSuccessStatusCode;
+
+            }
+            catch (Exception e)
+            {
+                Event(e, CreateLoggerArgs(EnumEventLevel.Error, $"CreateTestGroupAsync Failed - {e.Message}"));
+                throw;
+            }
+        }
+
+        public async Task<bool> UpdateAnalyteAsync(Analyte analyte, DateTime? effectiveDate = null)
+        {
+            var watch = System.Diagnostics.Stopwatch.StartNew();
+
+            var endpoint = $"/operations/sample/v1/analyte/{analyte.Id}";
+
+            if (effectiveDate.HasValue)
+            {
+                endpoint += $"?{effectiveDate.Value.ToString("O")}";

```

Comment:
The query parameter should have a name, not just a value. Should be:
$"?effectiveDate={effectiveDate.Value.ToString("O"}";